textproc/mdcat/Portfile

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        lunaryorn mdcat mdcat-0.19.0
```

Comment:
this is incorrect usage, now the version will be `mdcat-0.19.0`; instead you should use `github.setup        lunaryorn mdcat 0.19.0 mdcat-`.  The PG does `proc github.setup {gh_author gh_project gh_version {gh_tag_prefix ""} {gh_tag_suffix ""}}`.

Reply:
You're right - wasn't thinking here.  Change made.